swift/ql/src/queries/Security/CWE-757/InsecureTLS.ql

```diff
@@ -0,0 +1,55 @@
+/**
+ * @name Insecure TLS configuration
+ * @description TLS v1.0 and v1.1 versions are known to be vulnerable.. TLS v1.2 or v1.3 should be used instead.
+ * @kind path-problem
+ * @problem.severity error
+ * @security-severity TODO
```

Comment:
You can generate this value using the script in the [CWE-Scores](https://github.com/github/cwe-scores) repo.  There's instructions in the repo readme, but let me know if you have problems with that at all.